api/client/client_test.go

```diff
@@ -54,11 +53,8 @@ func TestNewDBFromAddr(t *testing.T) {
 	t.Run("test new db from address", func(t *testing.T) {
 		addr, err := ma.NewMultiaddr(info.Addresses[0])
 		checkErr(t, err)
-		fk, err := sym.FromBytes(info.FollowKey)
-		checkErr(t, err)
-		rk, err := sym.FromBytes(info.FollowKey)
```

Comment:
*Seems* that this also deleted a bug :P (`rk := ... info.FollowKey)`?)

Reply:
Oh, yep!